src/main/php/rdbms/mysql/MySQLConnection.class.php

```diff
@@ -99,7 +99,7 @@ public function connect($reconnect= false) {
       throw $e;
     }
 
-    mysql_query('set names UTF8', $this->handle);
+    mysql_query('set names utf8mb4', $this->handle);
```

Comment:
I'm not sure about this, the [PHP documentation](http://php.net/manual/en/function.mysql-set-charset.php) discourages using `set names` directly. The mysqli adapter uses the PHP function as well.